_scicomputing/access_methods.md

```diff
@@ -116,24 +116,25 @@ You will be prompted to enter and then confirm a passphrase. This is the best pr
 For extra points, read all about [public key cryptography](https://en.wikipedia.org/wiki/Public-key_cryptography) (it is very cool!).
 
 #### Key distribution
-
-You can copy your public key in any way you like. You can even cut-and-paste the key contents if that is easy. On Mac and Linux there is an auxiliary ssh command called `ssh-copy-key` which will copy your default public key to a specified remote system (asking for your password to make the ssh connect to transfer the key). On Windows it is easiest to cut-and-paste the key into a remote session opened using a password.
-
+You can copy your public key in any way you like. You can even cut-and-paste the key contents if that is easy. On Mac and Linux there is an auxiliary ssh command called `ssh-copy-id` which will copy your default public key to a specified remote system (asking for your password to make the ssh connect to transfer the key). 
+```
+ssh-copy-id HutchID@rhino
+```
+On Windows it is easiest to cut-and-paste the key into a remote session opened using a password.
 #### Passphrase management
 No one wants to type a long passphrase every time they use SSH. Creating an SSH key with no passphrase is unsafe. There must be a better way... .
 
 ##### MacOS Keychain
-
 The following steps will get your MacOS device into a state where your SSH key passphrase is unlocked for you when you log into the Mac, and is automatically supplied to SSH. You will need to be logged in to your Mac, have a terminal open, and have an SSH key to start.
 
+* Change the permissions of your public key `chmod 600 <path to keyfile>` the default is `~/.ssh/id_rsa.pub`. MacOS will not let you add your 'key' to the keychain without doing this first.
```

Comment:
Does OSX need the public key to be mode 600?  Typically this can be ugo+r...

Reply:
For some reason that wasn't working for me yesterday but is today. I will take this step out and leave permissions as default / 644 for the public key.